projects/WeTek_Hub/linux/linux.aarch64.conf

```diff
@@ -34,7 +39,7 @@ CONFIG_CROSS_COMPILE=""
 CONFIG_LOCALVERSION=""
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="@DISTRONAME@"
-CONFIG_SWAP=y
+# CONFIG_SWAP is not set
```

Comment:
I didn't say CONFIG_SWAP is needed, I said that it shouldn't be changed implicitly.